src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.iOS.cs

```diff
@@ -79,12 +79,19 @@ protected Size MeasureChildOverride(View view, Size slotSize)
 				ret.ToString();
 			}
 
+			if (!(view is FrameworkElement) && view is IFrameworkElement ife)
+			{
+				// If the child is not a FrameworkElement, part of the "Measure"
+				// phase must be done by the parent element's layouter.
+				// Here, it means adding the margin to the measured size.
+				ret = ret.Add(ife.Margin);
```

Comment:
What happens if the view is collapsed? Make sure that it doesn't set a non-zero measured size in that case.

Reply:
Aren't we not supposed to get here if the view is collapsed ?

Reply:
Ah, @jeromelaban is right, never mind!